modules/store-devtools/src/provide-store-devtools.ts

```diff
@@ -19,9 +19,14 @@ import {
   StoreDevtoolsOptions,
 } from './config';
 import { ReducerManagerDispatcher, StateObservable } from '@ngrx/store';
-import { createStateObservable } from './instrument';
 import { StoreDevtools } from './devtools';
 
+export function createStateObservable(
```

Comment:
To be inline with the rest this `method` should be moved after `IS_EXTENSION_OR_MONITOR_PRESENT`.

Reply:
I moved it after `createReduxDevtoolsExtension` so that the factory function definitions are in the same order as their usages in the `EnvironmentProvider` below.